test/psp.cpp

```diff
@@ -764,6 +802,27 @@ TEST_F(I64GnodeTest, test_9) {
     run(data);
 }
 
+
+TEST_F(I64GnodeTestImplicit, test_1) {
+
+    t_testdata data{
+        {
+            {{1_ts}},
+            {{0_ts, 1_ts}}
+        },
+        {
+            {{2_ts}},
+            {{0_ts, 1_ts}, {1_ts, 2_ts}}
+        },
+        {
+            {{3_ts}},
+            {{0_ts, 1_ts}, {1_ts, 2_ts}, {2_ts, 3_ts}}
+        }
+    };
+
+    run(data);
+}
```

Comment:
Please add a test to see what happens when someone sends in a tablet with psp_op + psp_pkey already set in to an implicitly pkeyed gnode